cypress/integration/BounceRulesDetailed/bounce_rules_detailed.spec.js

```diff
@@ -22,6 +22,7 @@ describe("Bounce Rule Detailed", () => {
     BounceRuleDetailedPage.teardownBounceRule(testCreateRule)
       .then(isTearDownSuccess => {
         if (isTearDownSuccess) {
+          cy.wait(1000);
```

Comment:
Making two CRUD calls back to back shouldn't break primary keys to the point of having to wait 1s between each call and cy.wait(someWaitMs) is more of a last resort. we should bring it up to the backend about this issue cause it's common to have multiple people using the app and making actions around the same time so something must be going on with the primary keys that needs to be refactored